tensorflow/core/distributed_runtime/rpc/grpc_worker_service.cc

```diff
@@ -337,32 +337,25 @@ void GrpcWorker::RecvTensorAsync(CallOptions* opts,
             if (src_dev->tensorflow_gpu_device_info() && (!on_host)) {
 #if GOOGLE_CUDA
               const DeviceContext* send_dev_context = send_args.device_context;
-              RecvTensorResponse* tmp = new RecvTensorResponse;
-              tmp->set_is_dead(is_dead);
+              AllocatorAttributes alloc_attrs;
```

Comment:
This should be interesting and I will take some time to look into the benchmarks. Thanks!